source/features/recently-pushed-branches-enhancements.tsx

```diff
@@ -54,7 +54,7 @@ async function init(): Promise<false | void> {
 	document.body.classList.add('rgh-recently-pushed-branches');
 
 	// Move or add list next to the notifications bell
-	select('.Header-item--full')!.after(widget);
+	select('.Header-item--full,.HeaderMenu nav')!.after(widget);
```

Comment:
`HeaderMenu` doesn't exist on regular GitHub, so no chances of conflict. If this works for you we can merge it